### ISSUE TITLE
Multiple timers

### ISSUE THREAD

**Issue by arquicanedo** [arquicanedo/barbecueboss#16]
Feature requested by user. Have multiple timers to track multiple steaks.

**Comment by arquicanedo** [comment]
Request originator pointed out to the [Multi Timer](https://apps.garmin.com/en-US/apps/d5a1a49f-3196-475b-ad61-eff15692ed14) app.

His idea as follows

> I would suggest being able to create a number of "steaks" with a name and a time (and maybe number of flips).
> Then on entering the app, there is a list of steaks (e.g. look at Multi Timer) each with a timer. On starting the timer you will see them all running (if there is space both total time and "this flip" time) and when it's time to flip, the screen is filled with a message and image of placement and the name of the one you need to flip.

**Comment by nilsbenson** [comment]
I'm thinking that popping up a view or alert showing which needs flipping and allowing the user to flip/restart the timer from it shouldn't be too hard. Would need to be able to queue new requests (as other meats finish) to display them but I don't think that's a problem.

**Comment by arquicanedo** [comment]
I haven't grilled with our latest version yet, but with the simulator I got the feeling that eliminating the flip confirmation makes it a better experience because it has less clicks. In the original version, I had to momentarily put the attention on the watch rather than on the steaks. These few seconds cost me burnt steaks a few times because the grill was flaming.

It will be interesting to compare the two approaches for alerting: no-flip vs flip+alarm screen. Let's try what works best. I agree with the queueing. Perhaps the alarm screen should display all the steaks that need flipping.

I think the no-flip version is interesting because when you're grilling it lets you plan ahead. For example, if I have a 30 seconds and a 2 minute expirations but a trip to the kitchen (inside the house), having all the timers in sight lets me plan and wait for 30 seconds, run inside to grab a beer, and come back for the 2 minutes. If the alarm pops up that may be a distraction. But at this point all is pure speculation. I think we can try it!
